sql/src/main/java/io/crate/protocols/postgres/ConnectionContext.java

```diff
@@ -329,19 +331,10 @@ private void closeSession() {
 
         @Override
         public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) throws Exception {
-            ignoreTillSync = true;
             try {
-                Messages.sendErrorResponse(ctx.getChannel(), e.getCause());
+                LOGGER.error("Uncaught exception: ", e.getCause());
```

Comment:
afaik the whole method does not make sense anymore as it only logs. also try/catch around the logger seems senseless to me